src/static/helpers/slasHelper.ts

```diff
@@ -39,11 +39,18 @@ export const getCodeAndUsidFromUrl = (
   };
 };
 
+const nanoid = (): string => {
+  const rng: PRNG = seedrandom(String(+new Date()), {entropy: true});
+  return customRandom(urlAlphabet, 128, size =>
+    new Uint8Array(size).map(() => 256 * rng())
+  )();
+};
```

Comment:
Consider a comment comment on why this random seeding is nessecary given that browsers should handle it.

Reply:
Done.